analyzer/src/main/kotlin/managers/Cargo.kt

```diff
@@ -28,6 +28,7 @@ import java.util.SortedSet
 
 import org.ossreviewtoolkit.analyzer.AbstractPackageManagerFactory
 import org.ossreviewtoolkit.analyzer.PackageManager
+import org.ossreviewtoolkit.analyzer.parseAuthorString
```

Comment:
Looks like now `@Suppress("TooManyFunctions")` is not needed anymore on this class. So maybe first introduce the global function and take it right away into use for Cargo instead of adding and removing the annotation back and forth?